search-commons/src/main/java/no/unit/nva/search2/ticket/TicketQuery.java

```diff
@@ -69,9 +73,22 @@ private TicketQuery() {
         assignImpossibleWhiteListFilters();
     }
 
+    @JacocoGenerated    // default value shouldn't happen, (developer have forgotten to handle a key)
```

Comment:
Hvis man er pedantisk så kan man si at switchen er da unødvendig, hvis default aldri kan skje. 

Forøvrig spesielt valg å gå for switch med bare én case som er reell også.